.sync.yml

```diff
@@ -46,6 +53,8 @@ Gemfile:
 
 # The rakefile is heavily modified for the build process
 Rakefile:
+  requires:
+  use_litmus_tasks: true
```

Comment:
if pdksync has been run we do not need this line. if it hasnt it is needed.

Reply:
removed litmus reference